<ui version="4.0" >
 <class>OptionsDialog</class>
 <widget class="QWidget" name="OptionsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>550</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Options</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="styleSheet" >
   <string/>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>640</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Fonts and colors</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>11</y>
           <width>415</width>
           <height>168</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="editorFontLabel" >
            <property name="text" >
             <string>Editor font:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="pbEditorColor" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="palette" >
               <palette>
                <active>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>208</red>
                    <green>161</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>208</red>
                    <green>161</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>208</red>
                    <green>161</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="toolTip" >
               <string>Editor text color</string>
              </property>
              <property name="autoFillBackground" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="editorFontComboBox" />
            </item>
            <item>
             <widget class="QSpinBox" name="editorSpinBox" />
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QCheckBox" name="editorBoldCheckBox" >
                <property name="text" >
                 <string>bold</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="editorItalicCheckBox" >
                <property name="text" >
                 <string>italic</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="statusbarFontLabel" >
            <property name="text" >
             <string>Statusbar font:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="pbStatusBarColor" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="palette" >
               <palette>
                <active>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>64</red>
                    <green>64</green>
                    <blue>64</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>64</red>
                    <green>64</green>
                    <blue>64</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>64</red>
                    <green>64</green>
                    <blue>64</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="toolTip" >
               <string>Status bar text color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFontComboBox" name="statusbarFontComboBox" />
            </item>
            <item>
             <widget class="QSpinBox" name="statusbarSpinBox" />
            </item>
            <item>
             <layout class="QHBoxLayout" >
              <item>
               <widget class="QCheckBox" name="statusbarBoldCheckBox" >
                <property name="text" >
                 <string>bold</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="statusbarItalicCheckBox" >
                <property name="text" >
                 <string>italic</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>111</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QPushButton" name="pbEditorBackColor" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize" >
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="palette" >
               <palette>
                <active>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="toolTip" >
               <string>Editor background color</string>
              </property>
              <property name="text" >
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Editor background color</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2" >
        <attribute name="title" >
         <string>Misc</string>
        </attribute>
        <widget class="QWidget" name="" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>18</y>
           <width>198</width>
           <height>221</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="fullScreenCheckBox" >
            <property name="text" >
             <string>Run TextRoom in fullscreen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="loadOnStartCheckBox" >
            <property name="text" >
             <string>Load last document on startup</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>30</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="saveCursorCheckBox" >
              <property name="whatsThis" >
               <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Flow Mode disables deletions from backspace, delete key and cuttings. This is done to help you focus on writing, instead of correcting minor typos. Typos and other editing can be done in a later stage.&lt;/p>&lt;/body>&lt;/html></string>
              </property>
              <property name="text" >
               <string>Restore cursor position</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="splashScreenCheckBox" >
            <property name="text" >
             <string>Show splash screen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autoSaveCheckBox" >
            <property name="text" >
             <string>Auto save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="flowModeCheckBox" >
            <property name="whatsThis" >
             <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Flow Mode disables deletions from backspace, delete key and cuttings. This is done to help you focus on writing, instead of correcting minor typos. Typos and other editing can be done in a later stage.&lt;/p>&lt;/body>&lt;/html></string>
            </property>
            <property name="text" >
             <string>Flow mode (no deletions)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>261</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>2</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pbEditorColor</tabstop>
  <tabstop>editorFontComboBox</tabstop>
  <tabstop>editorSpinBox</tabstop>
  <tabstop>editorBoldCheckBox</tabstop>
  <tabstop>editorItalicCheckBox</tabstop>
  <tabstop>pbStatusBarColor</tabstop>
  <tabstop>statusbarFontComboBox</tabstop>
  <tabstop>statusbarSpinBox</tabstop>
  <tabstop>statusbarBoldCheckBox</tabstop>
  <tabstop>statusbarItalicCheckBox</tabstop>
  <tabstop>pbEditorBackColor</tabstop>
  <tabstop>fullScreenCheckBox</tabstop>
  <tabstop>loadOnStartCheckBox</tabstop>
  <tabstop>splashScreenCheckBox</tabstop>
  <tabstop>autoSaveCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>531</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>361</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel" >
     <x>249</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadOnStartCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveCursorCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>269</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
